docs/pages/app-signing/apple-developer-program-roles-and-permissions.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Apple Developer Program roles and permissions for EAS Build
+sidebar_title: Apple Developer Program roles and permissions
+description: Learn about the Apple Developer account membership requirements for creating an EAS Build.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+
+An Apple Developer account is required to create iOS device builds on EAS. This account allows you to generate [app signing credentials](/app-signing/managed-credentials/#generating-app-signing-credentials) such as certificates, identifiers, and profiles, submit the app for review, and manage app's distribution.
+
+If the Apple Developer account is an individual account, only the account owner can generate the app signing credentials. This guide provides steps that the account owner has to follow to ensure app signing credentials are generated. It also provides steps for the team developer to create an EAS Build.
+
+> See [Apple's documentation on Program Roles](https://developer.apple.com/support/roles/) for details on the different roles and their permissions based on the type of Developer account.
+
+## Steps for Apple Developer account owner
```

Comment:
```suggestion
## Steps for Apple Developer with access to certificates, identifiers, and profiles
```
I dunno how often we want to repeat this, but this is only way I know how to describe this user.

Reply:
Yeah, I realize "owner" might not be a good word to use here. Looking https://developer.apple.com/support/roles/, majorly, there are four type of account roles can have access to different certificates and profiles. Let's go with "authorized users". Thanks for the suggestion!